sqlite3.Connection.DatabaseError
================================

.. currentmodule:: sqlite3

.. autoattribute:: Connection.DatabaseError